src/main.cpp

```diff
@@ -1304,11 +1301,11 @@ void setup() {
 
     editableVars["BREW_TIME"] = {.displayName = F("Brew Time (s)"),
                                  .hasHelpText = true,
-                                 .helpText = F("Stop brew after this time"),
+                                 .helpText = F("Stop brew after this time --> 0=Deactivated"),
```

Comment:
I don't like the --> notation, does not look very finished... 

How about "Stop brew after this time. Set to 0 to deactivate brew by time." ?